lib/SILOptimizer/Mandatory/DIMemoryUseCollector.cpp

```diff
@@ -223,6 +224,33 @@ SILType DIMemoryObjectInfo::getElementType(unsigned EltNo) const {
                            Module, MemorySILType, EltNo, isNonDelegatingInit());
 }
 
+/// During tear-down of a distributed actor, we must invoke its
+/// \p actorSystem.resignIdentity method, passing in the \p id from the
```

Comment:
Minor:

```suggestion
/// \p actorSystem.resignID method, passing in the \p id from the
```

meh sadly during review we were asked to stick to `ID` consistently :) 

C++ function names good to use Identity, that's fine :)